src/lib.rs

```diff
@@ -249,65 +282,68 @@ impl Dependencies {
         self.libs.iter().map(|(k, v)| (k.as_str(), v))
     }
 
-    fn aggregate_str<F: Fn(&Library) -> &Vec<String>>(
-        &self,
-        getter: F,
-    ) -> impl Iterator<Item = &str> {
-        self.libs
+    fn aggregate_str<F: Fn(&Library) -> &Vec<String>>(&self, getter: F) -> Vec<&str> {
+        let mut v = self
+            .libs
             .values()
             .map(|l| getter(l))
             .flatten()
             .map(|s| s.as_str())
-            .sorted()
-            .dedup()
+            .collect::<Vec<_>>();
+        v.sort();
+        v.dedup();
+        v
     }
 
-    fn aggregate_path_buf<F: Fn(&Library) -> &Vec<PathBuf>>(
-        &self,
-        getter: F,
-    ) -> impl Iterator<Item = &PathBuf> {
-        self.libs
+    fn aggregate_path_buf<F: Fn(&Library) -> &Vec<PathBuf>>(&self, getter: F) -> Vec<&PathBuf> {
+        let mut v = self
+            .libs
             .values()
             .map(|l| getter(l))
             .flatten()
-            .sorted()
-            .dedup()
+            .collect::<Vec<_>>();
+        v.sort();
+        v.dedup();
+        v
     }
 
     /// An iterator returning each [Library::libs] of each library, removing duplicates.
```

Comment:
Need to either update the doc or keep the existing API.

Reply:
(Same for the other methods).

Reply:
Also I just noticed that this is actually an API change. `impl Iterator` before, now `impl IntoIterator` (aka `Vec`).

Reply:
I'll update the documentation.